src/emoji/__tests__/emojiSelectors-test.js

```diff
@@ -71,3 +76,41 @@ describe('getAllRealmEmoji', () => {
     expect(getAllRealmEmoji(deepFreeze(state))).toEqual(expectedResult);
   });
 });
+
+describe('getActiveRealmEmojiTransformed and getAllRealmEmojiTransformed', () => {
+  test('object returned must have emoji names as the keys', () => {
+    const state = {
+      accounts: [{ realm: 'https://example.com' }],
+      realm: {
+        emoji: {
+          1: {
+            id: '1',
+            name: 'smile',
+            source_url: 'https://example.com/static/user_upload/smile.png',
+          },
+          2: {
+            id: '2',
+            name: 'laugh',
+            source_url: 'https://example.com/static/user_upload/laugh.png',
+          },
+        },
+      },
+    };
+
+    const expectedResult = {
+      smile: {
+        id: '1',
+        name: 'smile',
+        source_url: 'https://example.com/static/user_upload/smile.png',
+      },
+      laugh: {
+        id: '2',
+        name: 'laugh',
+        source_url: 'https://example.com/static/user_upload/laugh.png',
+      },
+    };
+
+    expect(getActiveRealmEmojiTransformed(deepFreeze(state))).toEqual(expectedResult);
+    expect(getAllRealmEmojiTransformed(deepFreeze(state))).toEqual(expectedResult);
```

Comment:
These should be separate tests. Very, very rarely use two `expect`s in the same test (never is ideal :).
Bundling them means that a single test will pass for many reasons. The goal of a test is to help us pinpoint faster what the issue is.
Testing multiple entities in the same test works against that.

Reply:
It is OK to reuse input data in order to make the multiple tests you necessarily create shorter and contain less repetition.

Reply:
Okay, will split this test into two shortly.

Reply:
I usually try to provide at least three tests per entity (sometimes much more):
* how it behaves when no data is yet available
* how does it behave with expected data
* what does it do with incorrect data